.github/workflows/ci.yaml

```diff
@@ -24,15 +24,13 @@ jobs:
   golangci-lint:
     name: golangci-lint
     runs-on: ubuntu-latest
-    env:
-      GOLANGCI_LINT_CONTAINER: golangci/golangci-lint:v1.23.2
     steps:
       - name: Check out code into the Go module directory
-        uses: actions/checkout@v1
-      - name: Pull golangci-lint docker container
-        run: docker pull ${GOLANGCI_LINT_CONTAINER}
-      - name: Run golangci-lint
-        run: docker run --rm -v $(pwd):/app -w /app ${GOLANGCI_LINT_CONTAINER} golangci-lint run
+        uses: actions/checkout@v2
```

Comment:
Please look at https://github.com/networkservicemesh/cmd-forwarder-sriov/pull/21
This file is part of the config that updates from cmd-template.

Let's figure out why https://github.com/networkservicemesh/cmd-forwarder-sriov/pull/21 is not applying

Reply:
#21 is not applying because ci job "Push docker image" failed. The reason of it is lack of runtime stage in Dockerfile.
PR with fix: #23